OneSky.Services/Util/Networking.cs

```diff
@@ -45,11 +45,22 @@ static Networking()
             
             R result;
 
-            try{
-                result = JsonConvert.DeserializeObject<R>(jsonResponse);
-            }catch{
-                throw new ArgumentOutOfRangeException($"Unable to convert web response to type: {typeof(R)}");
+            if (typeof(R) != typeof(string))
+            {
+                try
+                {
+                    result = JsonConvert.DeserializeObject<R>(jsonResponse);
+                }
+                catch
+                {
+                    throw new ArgumentOutOfRangeException($"Unable to convert web response to type: {typeof(R)}");
+                }
+            }
+            else
+            {
+                result = (R)Convert.ChangeType(jsonResponse, typeof(R));
```

Comment:
Since jsonResponse is already a string, is this ChangeType really necessary?

Reply:
Yes, this was my first route to return jsonReponse as a string, but the method has a response type of Type for R does not allow us to return as a string. I had to cast the response with Type in order for it to return correctly.